lib/src/parser/xref_reader.dart

```diff
@@ -157,7 +157,7 @@ class XRefReader {
           startIndex: (index[idx] as PDFNumber).toInt(),
           numEntries: (index[idx + 1] as PDFNumber).toInt(),
           entries: entries,
-          endIndex: (index[idx + 1] as PDFNumber).toInt()));
```

Comment:
This requires a more detailed check than I can do on mobile, was this caused by a bug you observed?

Reply:
@NicolaVerbeeck Yes, 
This line produced 2 subsections with the first startIndex = 0 and endIndex = 307
but the second had startIndex of 307 and the endIndex was 182 which was the total amount of the entries

Reply:
@zagronitay Should this then not be startIndex + numEntries aka: `index[idx] as PDFNumber).toInt()+index[idx+1] as PDFNumber).toInt()`